test/test_helper.rb

```diff
@@ -16,6 +16,12 @@ class ActionView::TestCase
   def setup_test_fixture
     @user = User.new(email: 'steve@example.com', password: 'secret', comments: 'my comment')
     @builder = BootstrapForm::FormBuilder.new(:user, @user, self, {})
+    # Simulate how the builder would be called from `form_with`.
+    if '5.1.0' <= ::Rails::VERSION::STRING && ::Rails::VERSION::STRING < '5.2.0'
+      @form_with_builder = BootstrapForm::FormBuilder.new(:user, @user, self, { skip_default_ids: true })
+    else
+      @form_with_builder = BootstrapForm::FormBuilder.new(:user, @user, self, {})
+    end
```

Comment:
You can spin up 5.1 form_with builder in the test that actually needs it instead of doing this in the setup.